lib/internal/modules/run_main.js

```diff
@@ -29,7 +29,7 @@ function resolveMainPath(main) {
 
 function shouldUseESMLoader(mainPath) {
   const userLoader = getOptionValue('--experimental-loader');
-  if (userLoader)
+  if (userLoader.length > 0)
```

Comment:
This is what was making most tests fail.

Reply:
🤦‍♂️ omg